applications/luci-app-upnp/po/ca/upnp.po

```diff
@@ -234,6 +238,63 @@ msgstr ""
 msgid "Value in KByte/s, informational only"
 msgstr "Valor en KByte/s, només per informació"
 
+#~ msgid ""
```

Comment:
This was an old translation key and now it's commented after you runned an i18n-update.
This happened because the key (e.g. original message) was changed.

The key was:
'ACLs specify which external ports may be **redirected** to which internal addresses and ports'
But the new is:
'ACLs specify which external ports may be **forwarded** to which internal addresses and ports'


The existing translation will be lost and translators from the Weblate won't see the old translation that they should just slightly change.
Instead you can change the translation key and then the existing translation will be kept. Yes, it will be outdated but it would be easier to slightly fix it.

In other places you just changed "UPNP" to "UPNP & NAT-PMP". This again made the strings not same and translation lost. But you can replace them in existing languages. The `&` will need for a translation (e.g. in Russian it would be `UPNP и NAT-PMP`) but you can leave it as `&` and that should be fine for now.

This would be a boring thing to do but you can use bulk replaces across files in your IDE.
We should keep the old translations because it may take entirety while someone will re-translate messages.

Reply:
Unless he's willing to augment every existing translation, this is the correct way to do it.